DemoApp/Sources/AppDelegate.swift

```diff
@@ -18,6 +18,10 @@ class AppDelegate: NSObject, UIApplicationDelegate, UNUserNotificationCenterDele
         UNUserNotificationCenter.current().delegate = self
         setUpRemoteNotifications()
         setUpPerformanceTracking()
+
+        // Setup a dummy video file to loop when working with from the simulator
+        InjectedValues[\.simulatorStreamFile] = Bundle.main.url(forResource: "test", withExtension: "mp4")
```

Comment:
The video is a bit large. I know it's not in the SDK, but still the whole repo's size increases. Can we somehow reduce its quality & size?